.github/workflows/build.yaml

```
@@ -108,9 +108,11 @@ jobs:
             - name: Build and push docker image
               uses: docker/build-push-action@v6
               with:
-                  push: true
+                  push: false
                   file: "./apps/${{ matrix.project }}/Dockerfile"
                   tags: ${{ vars.DOCKER_IMAGE_REGISTRY }}/${{ matrix.project }}:${{ steps.variables.outputs.PR_VERSION }}
+              env:
+                DEBUG: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)